cmux-tui/bindings/python/cmux/resources.py

```
@@ -4348,11 +4551,26 @@ def put(
         self,
         projection: Mapping[str, Any],
         *,
+        frontend_id: str,
+        window_id: str,
+        generation: str,
+        expected_projection_revision: Optional[str] = None,
         idempotency_key: Optional[str] = None, expected_revision: Optional[str] = None,
     ) -> MutationResult["FrontendProjection"]:
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)